htdocs/langs/en_US/banks.lang

```diff
@@ -148,7 +148,7 @@ BackToAccount=Back to account
 ShowAllAccounts=Show for all accounts
 FutureTransaction=Transaction in futur. No way to conciliate.
 SelectChequeTransactionAndGenerate=Select/filter checks to include into the check deposit receipt and click on "Create".
-InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value (such as, YYYYMM)
+InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value: YYYYMM or YYYYMMDD
```

Comment:
Hi daamien. We are able to accept other format than those. Maybe what should be change is numeric to alphanumeric, because that is the ordering criteria we are following